src/main/java/org/nustaq/serialization/FSTConfiguration.java

```diff
@@ -474,6 +474,8 @@ protected static FSTConfiguration initDefaultFstConfigurationInternal(FSTConfigu
         reg.putSerializer(BigInteger.class, new FSTBigIntegerSerializer(), true);
 
         reg.putSerializer(FSTUnmodifiableCollectionSerializer.UNMODIFIABLE_COLLECTION_CLASS, new FSTUnmodifiableCollectionSerializer(), true);
+        reg.putSerializer(FSTUnmodifiableCollectionSerializer.UNMODIFIABLE_LIST_CLASS, new FSTUnmodifiableCollectionSerializer(), true);
```

Comment:
@RuedigerMoeller I didn't want to change this without checking with you first but shouldn't the second argument be false since these classes shouldn't be sub-classed?